src/scenes/Activities/index.js

```diff
@@ -60,18 +60,18 @@ export default class Activities extends Component {
   }
 
   tabIdx = { // Determine tab index based on path
-    [set]: TAB_SET,
-    [finalize]: TAB_FINALIZE,
-    [withdraw]: TAB_WITHDRAW,
-    [activityHistory]: TAB_HISTORY,
+    [SET]: TAB_SET,
+    [FINALIZE]: TAB_FINALIZE,
+    [WITHDRAW]: TAB_WITHDRAW,
+    [ACTIVITY_HISTORY]: TAB_HISTORY,
   }[this.props.match.path]
 
   componentDidMount() {
     const locations = {
-      [set]: AppLocation.resultSet,
-      [finalize]: AppLocation.finalize,
-      [withdraw]: AppLocation.withdraw,
-      [activityHistory]: AppLocation.activityHistory,
+      [SET]: AppLocation.SET,
+      [FINALIZE]: AppLocation.FINALIZE,
+      [WITHDRAW]: AppLocation.WITHDRAW,
+      [ACTIVITY_HISTORY]: AppLocation.ACTIVITY_HISTORY,
     };
     const appLocation = locations[this.props.match.path];
```

Comment:
I believe we can actually do `const location = _.invert(AppLocation)[this.props.match.path]`

Reply:
Then we can remove `const locations = {` object

Reply:
updated